### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/uno-build-error-codes.md [hunk 1]
```diff
@@ -141,6 +141,14 @@ The project doesn't know how to run the profile with name 'MyApp (Desktop WSL2)'
 
 To fix the issue, reorder the items in your `.csproj` so that `TargetFrameworks` contains `netX.0-desktop` as the first target framework.
 
+The Uno Platform team is following this [Visual Studio issue](https://developercommunity.visualstudio.com/t/WSL-launch-profile-cannot-be-found-when/10776961).
+
+### UNOB0016: The Publish Profile file must not contain the "PublishDir"
+
+When publishing an app using Click Once on Windows, the `PublishProfile` file may not contain a `PublishDir` entry if the command line parameter `UnoClickOncePublishDir` has been set.
```
Comment (morning4coffe-dev) [inline doc/articles/uno-build-error-codes.md:148]:
```suggestion
When publishing an app using ClickOnce on Windows, the `PublishProfile` file may not contain a `PublishDir` entry if the command line parameter `UnoClickOncePublishDir` has been set.
```

File: doc/articles/uno-build-error-codes.md [hunk 2]
```diff
@@ -141,6 +141,14 @@ The project doesn't know how to run the profile with name 'MyApp (Desktop WSL2)'
...
+
+This situation is due to an MSBuild property overriding bug found in the .NET SDK. For more information, see our [publishing with Click Once](xref:uno.publishing.desktop#windows-clickonce) documentation.
```
Comment (morning4coffe-dev) [inline doc/articles/uno-build-error-codes.md:150]:
```suggestion
This situation is due to an MSBuild property overriding bug found in the .NET SDK. For more information, see our [publishing with ClickOnce](xref:uno.publishing.desktop#windows-clickonce) documentation.
```

File: doc/articles/uno-publishing-desktop.md [hunk 3]
```diff
@@ -43,6 +45,98 @@ dotnet publish -f net8.0-desktop -r {{RID}} -p:SelfContained=true
...
+
+To use the Visual Studio publishing wizard:
+
+- Select the `netX.0-desktop` target framework in the debugger drop-down
+- In the Solution Explorer, right click on your project then sekect **Publish**
+- Click the **+ New profile** button
+- Select **ClicOnce**, then **Next**
+- Configure your app publishing in all the following wizard pages
+- In the **Configuration** section, make sure to select **Portable** for the **Target runtime**
+- Click Finish.
```
Comment (morning4coffe-dev) [inline doc/articles/uno-publishing-desktop.md:120]:
```suggestion
- Select the `netX.0-desktop` target framework in the debugger drop-down
- In the Solution Explorer, right click on your project then select **Publish**
- Click the **+ New profile** button
- Select **ClickOnce**, then **Next**
- Configure your app publishing in all the following wizard pages
- In the **Configuration** section, make sure to select **Portable** for the **Target runtime**
- Click **Finish**.
```

File: doc/articles/uno-publishing-desktop.md [hunk 4]
```diff
@@ -43,6 +45,98 @@ dotnet publish -f net8.0-desktop -r {{RID}} -p:SelfContained=true
...
+
+***
+
+Once done, you can use the following command in your CI environment, or using a **Developer Command Prompt for Visual Studio**:
+
+```shell
+msbuild /m /r /target:Publish /p:Configuration=Release /p:PublishProfile="Properties\PublishProfiles\ClickOnceProfile.pubxml" /p:TargetFramework=net8.0-desktop
+```
+
+The resulting package will be located in the `bin\publish` folder. You an change the output folder using `/p:UnoClickOncePublishDir=your\output\directory`.
```
Comment (morning4coffe-dev) [inline doc/articles/uno-publishing-desktop.md:132]:
```suggestion
The resulting package will be located in the `bin\publish` folder. You can change the output folder using `/p:UnoClickOncePublishDir=your\output\directory`.
```

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.5

**Review by ajpinedam — APPROVED** [review]
Looks good

**Pull request merged**